octodns_route53/source.py

```diff
@@ -69,9 +72,14 @@ def instances(self):
                         key = tag['Key']
                         val = tag['Value']
                         if key == 'Name':
-                            fqdns.append(val)
+                            fqdns.append(val + self.tag_append)
                         elif key.startswith(self.tag_prefix):
-                            fqdns.extend(val.split('/'))
+                            fqdns.extend(
+                                [
+                                    fqdn + self.tag_append
+                                    for fqdn in val.split('/')
+                                ]
+                            )
```

Comment:
Square brackets aren't necessary here.

```suggestion
                            fqdns.extend(
                                    fqdn + self.tag_append
                                    for fqdn in val.split('/')
                            )
```

Reply:
Fixed in 48bb5e7